src/twisted/web/twcgi.py

```diff
@@ -155,10 +157,10 @@ class FilteredScript(CGIScript):
     """
     I am a special version of a CGI script, that uses a specific executable.
 
-    This is useful for interfacing with other scripting languages that adhere to
-    the CGI standard. My C{filter} attribute specifies what executable to run,
-    and my C{filename} init parameter describes which script to pass to the
-    first argument of that script.
+    This is useful for interfacing with other scripting languages that adheres
```

Comment:
Nitpick: this introduces a grammatical disagreement.  "scripting language*s* that adhere to" *or* "scripting language that adhere*s* to" but not this.